docs/dev/new_check_howto.md

```diff
@@ -300,9 +300,59 @@ The `logos` folder must contain **three** images with filenames and sizes that m
+| `manifest_version`  | String          | Mandatory          |                                                                                                                                                                                                                                    |
+| `name`              | String          | Mandatory          | Unique name for the integration. Use the folder name for this parameter.                                                                                                                                                           |
+| `public_title`      | String          | Mandatory          | Title of the integration displayed on the documentation. Should follow the following format: `Datadog-<INTEGRATION_NAME> Integration`                                                                                              |
+| `short_description` | String          | Mandatory          | This text -Maximum 80 characters- appears at the top of the integration tile as well as the integration's rollover text on the integrations page.                                                                                  |
+| `support`           | String          | Mandatory          |                                                                                                                                                                                                                                    |
+| `supported_os`      | Array of String | Mandatory          | List of supported OSs. Choose between `linux`,`mac_os`, and `windows`.                                                                                                                                                             |
+| `type`              | String          | Mandatory          | Type of the integration, should be set to `check`.                                                                                                                                                                                 |
+| `aliases`           | Array of String | Optional           | A list of URL aliases for the Datadog documentation.                                                                                                                                                                               |
+| `description`       | String          | Optional           | This text appears when sharing an integration documentation link.                                                                                                                                                                  |
+| `is_beta`           | Boolean         | Optional           | Default `false`. If set to `true` the integration `README.md` content is not displayed in the Datadog public documentation.                                                                                                        |
```

Comment:
What's the difference with `is_public` ?

Reply:
@l0k0ms ^